chromatic/.storybook/stories/bingel-dc.stories.ts

```diff
@@ -0,0 +1,37 @@
+import { All as AllButton} from "../../../packages/components/button/src/button.stories";
+import { All as AllCheckbox} from "../../../packages/components/checkbox/src/checkbox.stories";
+import { All as AllRadioGroup} from "../../../packages/components/radio-group/src/radio-group.stories";
+import { html } from 'lit';
+import { setPseudoStates, setTheme } from "../../utils/theme"
+
+export default {
+  title: 'Bingel DC',
+  decorators: [
+    (Story) => {
+      return html`
+        ${setTheme('bingel-dc')}
```

Comment:
Maybe you could define the decorator in 1 place (`preview.ts`), and then have `'bingel-dc'` as an argument/parameter to the story????